lib/roqua/support/errors.rb

```diff
@@ -93,15 +94,22 @@ def self.notify_appsignal(exception, parameters = {})
               Appsignal::Transaction.current.set_tags(parameters)
               Appsignal::Transaction.current.add_exception(exception)
             else
-              transaction = Appsignal::Transaction.create(SecureRandom.uuid, ENV.to_hash)
+              transaction = Appsignal::Transaction.create(SecureRandom.uuid, transaction_type)
```

Comment:
Roqua zit op 2.0.4, die versie wil hier 3 argumenten hebben.
Stiekem gebruiken we deze code nergens binnen roqua denk ik, omdat we altijd of binnen een een webrequest zitten of binnen een delayed_job, waarbij we in de if-tak zitten.

Denk wel dat als we in deze tak komen dat we er vanuit kunnen gaan dat we in een background-task zitten, dus hoef je niet perse mee te geven. 

`Appsignal::Transaction.create(SecureRandom.uuid, Appsignal::Transaction::BACKGROUND_JOB, Appsignal::Transaction::GenericRequest.new({}))`

Al hebben we dus een andere oplossing nodig voor het feit dat we maar 1 melding binnen krijgen voor de hele prepare_subscriptions job. Mogelijk door een vlag new_transaction mee te geven ofzo (of iets minder appsignal-specifieks met die strekking).